crates/bitwarden-error-macro/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+mod args;
+mod attribute;
+mod basic;
+mod flat;
+mod full;
+
+#[proc_macro_attribute]
+pub fn bitwarden_error(
```

Comment:
We should add a doc comment here explaining what this macro does and what each of the supported variants mean, that way it should be visible in the user's IDE.

Reply:
Thanks for the reminder!